src/nxdoc/jsf-ui-to-web-ui.md

```diff
@@ -32,7 +32,7 @@ Tutorials have been created to let you discover the basics of Web UI configurati
 - [HOWTO: Customize Document Layouts]({{page version='' space='nxdoc' page='web-ui-document-layouts'}})
 - [HOWTO: Customize Workflow Tasks]({{page version='' space='nxdoc' page='web-ui-workflow-tasks'}})
 - [HOWTO: Configure Searches]({{page version='' space='nxdoc' page='web-ui-search'}})
-- [HOWTO: Insert a New Pill]({{page version='' space='nxdoc' page='how-to-new-pill'}})
```

Comment:
This link is also referenced on the static branch side and will need to be updated on another PR.